lib/rails_admin/adapters/active_record.rb

```diff
@@ -30,7 +30,17 @@ def all(options = {}, scope = nil)
         scope = scope.includes(options[:include]) if options[:include]
         scope = scope.limit(options[:limit]) if options[:limit]
         scope = scope.where(primary_key => options[:bulk_ids]) if options[:bulk_ids]
-        scope = query_scope(scope, options[:query]) if options[:query]
+
+        if options[:query]
+          begin
+            # if you're using pg_search, the method is not defined
+            # eventhough it's there
+            scope = scope.rails_admin_search(options[:query])
```

Comment:
The search method's name should be configurable via DSL.

Reply:
For the time being, it's good to have this one;
I'll work on making it configurable from Rails Admin's model config later on.

Let me know if you wish me to proceed with the second one before pulling.

-- Btw, this little fix saved me hours and hours of building a custom search page

Reply:
Configurability is a key part of RailsAdmin, I don't merge this without it.
Also, having tests to make sure that this feature works is mandatory.